spec/features/notifications/notification_center/split_screen_spec.rb

```diff
@@ -67,7 +67,7 @@
       split_screen.switch_to_tab tab: "relations"
       split_screen.expect_tab :relations
       relations_tab = Components::WorkPackages::Relations.new(work_package)
-      relations_tab.expect_no_relation work_package
+      relations_tab.expect_no_relations
```

Comment:
@ulferts this flickering spec is addressed [in another PR](https://github.com/opf/openproject/pull/18530), I think we can safely ignore it here.

Reply:
@dombesz , I would still, independent from the fixes in the other PR do those here as the helper is misused. The line currently tests that the work package for which the relations are looked at is not listed. In other words, it ensures, that the work package is not related to itself. That cannot be the intend of the spec.